client/modules/state/state.go

```diff
@@ -18,13 +15,8 @@ const (
 	OperationsKey        = "operations"
 	DeletedOperationsKey = "deleted_operations"
 	FSMStateKey          = "fsm_state"
-	SignaturesKeyPrefix  = "signatures"
```

Comment:
you have moved SignaturesKeyPrefix to the signatures repo, but why you haven't moved OperationsKey and DeletedOperationsKey to the operations repo? you have made a copy which is not good, cause now we have two constants meaning the same in two different packages